src/blister.h

```diff
@@ -352,6 +355,147 @@ extern void ConvBlist (
 
 /****************************************************************************
 **
+*F  CopyBits( <fromblock>, <from-starting-bit>, <toblock>, <to-starting-bit>,
+**            <numbits> )
+**
+**  `CopyBits' copies <numbits> bits (numbering bits within a UInt
+**   from the least significant to the most significant) starting with
+**   bit number <from-starting-bit> of UInt *<fromblock> to a destination
+**   starting at bit <to-starting-bit> of *<toblock>.
+**
+**   The full function is provided as an inline function here because it is
+**   used in both blister.c and vecgf2.c and the compiler can do significant
+**   optimisations if some or all of <frombits>, <tobits> and <numbits> are
+**   known at compile time.
+*/
+
+/* constructs a mask that selects bits <from> to <to> inclusive of a UInt */
+static inline UInt mask(UInt from, UInt to)
```

Comment:
Hm, `mask` is an awfully common name, not sure it's a good idea to use that for a global function.